src/l500/l500-depth.cpp

```diff
@@ -379,6 +458,19 @@ namespace librealsense
        return l500::max_usable_range(noise_estimation);
     }
 
+    stream_profiles l500_depth_sensor::get_debug_stream_profiles()
+    {
+        stream_profiles results;
+
+        for( auto p : *_debug_profiles )
```

Comment:
`_debug_profiles` is just for caching the results? Any reason not to simply call `init_debug_stream_profiles` every time?

Reply:
why not to cache it like sensor_base:: _profiles ?